crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1098,7 +1098,11 @@ impl RenderMeshInstanceGpuBuilder {
                     mesh_vertex_slice.range.start,
                     mesh_vertex_slice.range.end - mesh_vertex_slice.range.start,
                 ),
-                None => (0, 0),
+                None => {
+                    // GPU memory for this mesh hasn't been allocated yet. Retry next frame.
+                    meshes_to_reextract_next_frame.insert(entity);
```

Comment:
If this is the fix, we might need to do the same for the `mesh_index_slice` a few lines below this.

I'm unsure why it previously didn't return `None` in this case. I feel like this might not be the full fix. I'll need to investigate further.

Reply:
Oh, thinking about this more, no, we shouldn't do this for the index_slice because not having indices is a valid case

Reply:
Is there a case where vertex_slice is ready, but not index_slice?

Reply:
For non indexed meshes index_slice will never be ready. I'm not sure it's possible for an indexed mesh to not have the index_slice ready. AFAIK they are allocated together but I could be wrong.